docs/API.md

```diff
@@ -51,19 +79,35 @@ Empty for now.
 - `Operation.type()` : string - Returns either `ClientSubscribing`, `ClientPublishing`, `ApplicationSubscribing`, `ApplicationPublishing`
 
 ### Channel
+- `Channel.name()` : string (outcome of https://github.com/asyncapi/markdown-template/blob/master/components/Channels.js#L26)
```

Comment:
```suggestion
```
We decided to use `Channel.path()` to represent this value, or is `name` more fitting?

Reply:
Oh you are right. TBH I think It's simpler if we keep name instead.

Reply:
The way it is defined in the spec is:
https://www.asyncapi.com/docs/specifications/2.0.0#channelsObject
>Holds the relative paths to the individual channel and their operations. Channel paths are relative to servers. Channels are also known as "topics", "routing keys", "event types" or "paths".
> A relative path to an individual channel. The field name MUST be in the form of a RFC 6570 URI template. 

I still think `path` is more suiting, since `name` is more vague and does not really explain what it is 🤔

Reply:
Fair enough. Removed `name()` in favor of `path()`. Changes were also applied to the [Parser branch](https://github.com/jonaslagoni/parser-js/tree/feature/mock_implementation) and to my [markdown-template branch](https://github.com/asyncapi/markdown-template/compare/master...smoya:feature/intent_api)